modules/ROOT/pages/audit-log-retention.adoc

```diff
@@ -0,0 +1,40 @@
+= Audit Log Retention Period
+
+xref:audit-logging.adoc[Audit logs] have a default retention period of six years. Users who have Audit Log Config Manager and Organization Administrator permissions can configure the retention period for the audit logs in their organization.
+
+Download your logs periodically if you want to maintain your log files for longer than six years.
+To check the audit log retention period for your organization, check the retention period in your root organization's settings, or use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/minor/1.0/console/method/%23335/[Audit Logging API^].
+
+When you configure a retention period:
+
+* You can specify a retention period between 30 and 2190 days.
+* The date the new retention period takes effect must be a minimum of seven days from the current date.
+* Configuring a new retention period affects your organization's compliance policy.
+
+== Before You Begin
+
+Before getting started, ensure that you have an account that has the following permissions:
+
+* Audit log config manager
+* Organization administrator (required only if you are using the user interface to set retention. You do not need this permission if you are using the API.)
+
+Your organization must also have *Try new features* or the Teams feature enabled in Access Management. 
```

Comment:
It seems strange to me that "Try new features" is bolded and "Team feature" is not bolded.  Even if *Try new features* ise an item in the GUI, I don't think it should be bolded here because you are not talking about the GUI here. Up to you.